packages/flet/lib/src/utils/theme.dart

```diff
@@ -159,7 +160,7 @@ ThemeData parseTheme(
     checkboxTheme: parseCheckboxTheme(value?["checkbox_theme"], theme),
     radioTheme: parseRadioTheme(value?["radio_theme"], theme),
     badgeTheme: parseBadgeTheme(value?["badge_theme"], theme),
-    switchTheme: parseSwitchTheme(value?["switch_theme"], theme),
```

Comment:
Instead of passing both `theme` and `context`, we could pass just `context` and derive `theme` from it while in the handlers.